src/relay/transforms/partition_graph.cc

```diff
@@ -404,21 +404,88 @@ IRModule RemoveDefaultAnnotations(IRModule module) {
   return module;
 }
 
+/*! \brief There can be regions with multiple outputs where each output
+ *  could be a tuple output. Such tuple outputs needs to be flattened
+ *  otherwise the function would create tuples of tuples.
```

Comment:
Might be worth explicitly mentioning this isn't supported by the graph runtime or relay VM, as it is otherwise valid relay.

Reply:
done!